manifests/w/Wesnoth/BattleForWesnoth/1.17.4/Wesnoth.BattleForWesnoth.installer.yaml

```diff
@@ -24,11 +24,13 @@ Installers:
   InstallerSha256: B1A2822AE0763426CE52026B07C438C786203CE580DB460B92D5EDCDCABE6B8C
   InstallerSwitches:
     Custom: /CURRENTUSER
+  ProductCode: Battle for Wesnoth 1.17.4
```

Comment:
Should we do a globally scoped ProductCode since we're doing a globally scoped ARP entries?

Reply:
I think we could, but WingetCreate puts everything on the installer node, so I was trying to not cause too much disruption